cargo-dist/Cargo.toml

```diff
@@ -36,14 +36,11 @@ axocli = { version = "0.2.0", optional = true }
 axotag = "0.1.0"
 cargo-dist-schema = { version = "=0.11.1", path = "../cargo-dist-schema" }
 
-axoasset = { version = "0.7.0", features = ["json-serde", "toml-serde", "toml-edit", "compression"] }
+axoasset = { git = "https://github.com/axodotdev/axoasset", branch = "decompress", features = ["json-serde", "toml-serde", "toml-edit", "compression"] }
```

Comment:
i assume we won't merge this until we publish the axo asset version with this feature?

Reply:
Made that release and swapped over to it!